src/Terminal/JupyterLabAppFactory.ts

```diff
@@ -34,11 +46,12 @@ export class JupyterLabAppFactory {
     const session = await manager.startNew();
     session.messageReceived.connect(async (_, message: IMessage) => {
       const content = message.content && message.content[0]?.toString();
-      if (this.checkShellClosed && message.type == "stdout") {
+      console.log(content);
```

Comment:
Remove

Reply:
so the new terminal line (cosmosuser@...) is not seen once the shell is launched. If is only seen next after it shuts down or crashes in between.